_learning_hours/devops/github_actions.md

```diff
@@ -0,0 +1,190 @@
+---
+theme: devops
+title: Basic GitHub Actions
+kata: fizzbuzz
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+difficulty: 1
+author: balp
+affiliation: HiQ
+---
+
+# Basic GitHub Actions
+
+## Learning Outcome
+
+* The students should have basic familiarity with GitHub
+  actions and a simple CI flow.
+
+## Requirements
+
+* For each pair
+    * Computer, internet connection
+    * Web-browser
+    * GitHub account
+* Basic YAML knowledge
+* Basic knowledge about CI
+
+
+## Session Outline
+
+* 7 min: Code review of rust code
+* 7 min: GitHUb actions and environment basics
+* 40 min: Make your CI flow
+* 5 min: Explain the main idea with a CI Flow
+
+  
+### Connect: Get a basic understanding on the rust library 
+
+A quick code review of [lib.rs](src/lib.rs)
```

Comment:
I think this link will need to be more fully qualified and point to github somewhere?